app/src/main/java/org/p2p/wallet/lokalise/LokaliseManager.kt

```diff
@@ -0,0 +1,36 @@
+package org.p2p.wallet.lokalise
+
+import android.content.Context
+import com.lokalise.sdk.BuildConfig
+import com.lokalise.sdk.Lokalise
+import com.lokalise.sdk.LokaliseCallback
+import com.lokalise.sdk.LokaliseUpdateError
+import timber.log.Timber
+
+private const val TAG = "LokaliseManager"
+
+object LokaliseManager {
+    fun setup(context: Context, lokaliseToken: String, projectId: String) {
+        with(Lokalise) {
+            init(context, lokaliseToken, projectId)
+            isPreRelease = BuildConfig.DEBUG
+            logsEnabled = BuildConfig.DEBUG
+            if (BuildConfig.DEBUG) {
+                addCallback(object : LokaliseCallback {
+                    override fun onUpdateNotNeeded() {
+                        Timber.tag(TAG).i("No update needed")
+                    }
+
+                    override fun onUpdateFailed(error: LokaliseUpdateError) {
+                        Timber.tag(TAG).i("Error on fetching locale $error")
```

Comment:
if LokaliseUpdateError is Throwable, use Timber.tag(TAG).i(error,"Error on fetching locale $error")